### DISCUSSION TITLE
USA NETWORK

### DISCUSSION THREAD

**Discussion by spotanjo3** [iptv-org/community#778] [Q&A]
### Channel Name

USA NETWORK

### Channel ID

USANetworkEast.us

### Website

https://www.usanetwork.com

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19754
